tutorials/transparent-proxy-and-filtering-on-k8s/index.md

```diff
@@ -0,0 +1,248 @@
+
+## Checkout the source repository
+
+1. Open [Cloud Shell](https://console.cloud.google.com/cloudshell)
+
+2. Clone the repository containing the code for this tutorial:
+
+        git clone https://github.com/danisla/kubernetes-tproxy
+        cd kubernetes-tproxy
+
+    The remainder of this tutorial will be run from the root of the cloned repository directory.
+
+## Create Container Engine cluster and install Helm
+
+1. Create the Container Engine cluster:
+
+        gcloud container clusters create tproxy-example --zone us-central1-f
+
+    This command will also automatically configure the kubectl command to use the cluster.
+
+2. Install [Helm](https://github.com/kubernetes/helm) locally in your Cloud Shell instance:
+
+        curl -sL https://storage.googleapis.com/kubernetes-helm/helm-v2.5.1-linux-amd64.tar.gz | tar -xzvf - && sudo mv linux-amd64/helm /usr/local/bin/ && rm -Rf linux-amd64
+
+3. Initialize Helm:
+
+        helm init
+
+    This will install the server side component of Helm, Tiller,  in the Kubernetes cluster. The Tiller pod may take a minute to start, run the command below to verify it has been deployed:
+
+        helm version
+
+    You should see the Client and Server versions in the output:
+
+        Client: &version.Version{SemVer:"v2.5.1", GitCommit:"7cf31e8d9a026287041bae077b09165be247ae66", GitTreeState:"clean"}
+        Server: &version.Version{SemVer:"v2.5.1", GitCommit:"7cf31e8d9a026287041bae077b09165be247ae66", GitTreeState:"clean"}
+
+## Install the Helm chart
+
+Before installing the chart, you must first extract the certificates generated by mitmproxy. The generated CA cert will be used in the example pods to trust the proxy when making HTTPS requests.
+
+1. Extract the generated certs using Docker:
+
+        cd charts/tproxy
+        docker run --rm -v ${PWD}/certs/:/home/mitmproxy/.mitmproxy mitmproxy/mitmproxy >/dev/null 2>&1
+
+2. Install the chart:
+
+        helm install -n tproxy .
+
+    The output of this command shows you how to augment your deployments to use the init container and trusted certificate configmap volume. Example output below:
+
+        Add the init container spec below to your deployments:
+
+        initContainers:
+        - name: tproxy
+        image: docker.io/danisla/tproxy-sidecar:0.1.0
+        imagePullPolicy: IfNotPresent
+        securityContext:
+            privileged: true
+        env:
+        resources:
+            limits:
+            cpu: 500m
+            memory: 128Mi
+            requests:
+            cpu: 100m
+            memory: 64Mi
+
+        Add the volumes below to your deployments to use the trusted https tproxy:
+
+        volumes:
+        - name: ca-certs-debian
+        configMap:
+            name: tproxy-tproxy-root-certs
+            items:
+            - key: root-certs.crt
+            path: ca-certificates.crt
+
+
+3. Run the command below to get the status of the DaemonSet pods
+
```

Comment:
You don't need, "Run the command below". You can just say, "Get the status of the DaemonSet pods:".